.circleci/config.yml

```diff
@@ -1,16 +1,12 @@
 version: 2.1
 
 executors:
-  default:
+  terraform:
     docker:
-      - image: circleci/node
-  docker-awscli:
-    docker:
-      - image: ussba/cc-docker-git-aws
+    - image: hashicorp/terraform:light
   docker-poirot:
     docker:
-      - image: ussba/poirot:3
-
+      - image: ghcr.io/ussba/sba-containers/poirot:3.9
```

Comment:
Do we really need to PR & merge for a test? I reckon we can test it out fine enough by just adding it to your branch and running tests on the branch.